scripts/ci/test_index.py

```diff
@@ -21,6 +21,10 @@
 
 from util import get_ext_metadata, get_whl_from_url, get_index_data, verify_dependency
 
+# Some extensions are published through manually submitted PRs and the latest version may not be in the end of
+# the version list in index.json. Let's just test all versions for these extensions.
+EXTS_TEST_ALL_VERSIONS = {'azure-cli-ml', 'alias', 'azure_iot'}
```

Comment:
Maybe we can sort all versions and only test the latest one?

Reply:
Python's built-in function [`max`](https://docs.python.org/3/library/functions.html#max) supports a custom key extraction function and we can use [`packaging.version`](https://packaging.pypa.io/en/latest/version.html) for version comparison.

Reply:
updated with picking a `max` version so the order of versions does not matter now.